docs/faq.md

```diff
@@ -0,0 +1,29 @@
+# Frequently Asked Questions
+
+## Querier
+
+#### How does Thanos Query show the user the same metric values from the two Prometheus instances?
+
```

Comment:
What do you think about the following? I'd also wonder should we expand on the point about the availability of the object storage?

```suggestion
Querier supports this via two different modes of deduplication!

Online deduplication - Querier transparently deduplicates metrics from replicated sources such as Prometheus replicas, replicated ingestion via Receivers and more. You can read more about online deduplication in the [Querier component docs](components/query.md).

Offline deduplication - This occurs inside [Compactor Vertical Compaction](components/compact.md#vertical-compactions) logic which offers deduplication of data at rest. This can be used when you want to reduce your storage size (make sure your object storage is available enough you can afford no back up of your data!). When offline deduplication is enabled, Querier will have no duplicated data to deduplicate.
```

Reply:
Ack, will take a look this week